src/components/BookItem/BookItem.js

```diff
@@ -12,21 +13,28 @@ const BookItem = ({ ID, book }) => {
   };
 
   return (
-    <div>
-      <li>
-        Title:
-        {' '}
-        {title}
-        {' '}
-        Author:
-        {' '}
-        {author}
-        {' '}
-        Category:
-        {' '}
-        {category}
-      </li>
-      <button type="button" onClick={handleRemove}>delete</button>
+    <div className="book-container">
+      <div className="book-layout">
+        <div className="book-imfos">
+          <div className="category">{category}</div>
+          <div className="title">{title}</div>
+          <div className="author">{author}</div>
+          <button type="button" onClick={handleRemove} className="delete">delete</button>
+        </div>
```

Comment:
- [ ] Kindly add other place holders like the `edit` and `comment` beside the remove button and make them have the same colors and design with the one in the design guideline so that your design matches that of the design guideline. 

Your Implementation | Zeplin Design
:---:|:---:
![](https://i.imgur.com/BMCHc46.png) | ![](https://i.imgur.com/plncrnm.png)